src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> listData;
+        try {
+            File file = new File(fromFileName);
+            listData = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + fromFileName, e);
+        }
+        File file = new File(toFileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file: " + toFileName, e);
+        }
+        try {
+            Files.write(file.toPath(), calculatedData(listData).getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file: " + toFileName, e);
+        }
+    }
 
+    private String calculatedData(List<String> listData) {
+        StringBuilder result = new StringBuilder();
+        String[] supplyBuy;
+        int sumSupply = 0;
+        int sumBuy = 0;
+        for (String line : listData) {
+            supplyBuy = line.split(",");
+            if ("supply".equals(supplyBuy[0])) {
```

Comment:
"supply" -magic value
0 - magic value
please check the other places for magic values and create an appropriate constants.